app/Providers/AppServiceProvider.php

```diff
@@ -16,7 +20,12 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+       $this->app->singleton(RequestLoggerInterface::class, function (){
+           return $this->app->make(DebugRequestLogger::class);
```

Comment:
Should be implmentnted check for which logger to provide.

Reply:
Am facut in al 2-lea commit